README.md

```diff
@@ -1,5 +1,208 @@
+
+#### CLI tool: Manual run (not devops pipeline)
+
+1. Download CLI tool component from [https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli](https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli). Configure the CLI for your GIT repository containing API specifications (YAML files).
+   
+```sh
+npm i @sinetiq/api-specification-registry-cli
+```
+
+#### CLI tool: Automation, devops (CI/CD) pipeline
+
+1. Use `pipeline example` matching your environment.
+
+2. Create the API specification and version control it in your git-repo, attached to your created pipeline.
+
+3. Create a Pull Request/Merge Request, review, approve and merge the API specification, initiate the pipeline.
+
+
+_Your demonstration environment is now ready._
+
+#### Verify installation
+
+1. Verify the API Specification Registry.  
+   Open a webbrowser, use url http://localhost:3333/ to verify up-n-running.
+
+2. Verify Service Registry.  
+   Open a webbroser, use url http://localhost:8500/ to verify up-n-running.
+
+3. Verify Service Explorer.  
+   Open Visual Code, verify the `Service Explroer` accroding to user manual.
+
+4. Verify CLI-installed.
+   ```sh
+   npm -v @sinetiq/api-specification-registry-cli
+   ```
+
+<p>&nbsp;</p>
+
+## Usage
+
+The steps 1 to 5 will show how to run this demonstration and successfully mange `governance of API specifications` in the complete life cycle chain.
+
+<div style="text-align:center">
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes-steps.png" width="600">
+</div>
+<p>&nbsp;</p>
+
+### Step 1: Create API Specification
+Use your preferred editor to create a YAML file for the API specification and store it in your versioning repository (ex. git). Ensure each specification has an API identifier (`x-api-identifier`) with a unique name. _The API specifications will be loosely coupled, no server part and only one specific information object (resource)._ 
+
+  ```sh
+    openapi: 3.0.0
+    info:
+      description: This is a simple API for accessing temperature sensor data.
+      title: Temperature Sensor API HTTP(S)-JSON
+      version: 1.0.0
+      x-service-identifier: temperature-sensor
+    paths:
+      /temperature:
+        get:
+          responses:
+            200:
+              content:
+                application/json:
+                  schema:
+                    $ref: '#/components/schemas/TemperatureData'
+              description: Temperature data
+          summary: Get current temperature data
+          x-accepts: application/json
+  ```
+
+### Step 2: Publish API Specification
+Utilize the approval CLI Tool to publish your API specification to the API specification repository. _Alternative: Use example pipelines matching your environment_.
+For test and deemonstration purpose use the command line call publish the specification.
+
+  ```sh
```

Comment:
Add `--registry localhost:3333` to use local registry instead of global

Reply:
Actually, we need to change the entire command. This command uses the glob pattern that searches for JSON files with a special data structure.

For demo purposes, it is easier to simply pass the OpenAPI directly and the parameters to the CLI, e.g.

`npx @sinetiq/api-specification-registry-cli -r localhost:3333 -g com.example -a temperature-rest-json -v 1.0.0 --ignore-already-registered`